test/simple_form/cases/helper.rb

```diff
@@ -18,6 +18,14 @@ def custom_wrapper_with_full_error
   end
 end
 
+def custom_wrapper_with_nested_error
+  SimpleForm.build tag: :div, class: :input, error_class: :field_with_errors do |b|
+    b.wrapper tag: :div do |b_nested|
+      b_nested.use :error, wrap_with: { tag: :span, class: :error }
+    end
```

Comment:
This is a similar configuration to what I have defined in my team's codebase. The main point here being that `:error` is nested inside a wrapper.